src/foam/u2/wizard/StepWizardletStepsView.js

```diff
@@ -294,13 +294,28 @@ foam.CLASS({
       code: async function() {
         var el    = await this.parentNode.el();
 
+        if ( ! this.data.currentWizardlet ) {
+          console.warn('setScrollPos() failed; no current wizardlet', {
+            data: this.data
+          });
+          return;
+        }
+
         var currChild = null;
         for ( let node of this.stepElements ) {
           if ( node.getAttribute('data-wizardlet') == this.data.currentWizardlet.id ) {
             currChild = node;
             break;
           }
         }
+        if ( ! currChild ) {
+          // TODO: investigate why this is called before elements are loaded
+          console.warn('setScrollPos() failed; unable to find active wizardlet element', {
```

Comment:
As above: Is this a fatal error for the wizard? Should be be throwing an error instead of just returning?

Reply:
This definitely shouldn't be a fatal error because it happens in the incremental wizard right now. It won't cause any issues but I put a `TODO` there 'cause it would be nice to know why.